### ISSUE TITLE
When using tab to open tooltip, the tooltip did not show in the correct position

### ISSUE THREAD

**Issue by jingx-xue** [carbon-design-system/carbon#3186]
## Environment

> Operating system
Windows
> Browser
Chrome
> Automated testing tool and ruleset
NA
> Assistive technology used to verify
NA
## Detailed description

> What version of the Carbon Design System are you using?
latest
> What did you expect to happen?
the tooltip content should be right below the icon when I hit enter
> What happened instead?
the tooltip content is position on the really left of the icon
> What WCAG 2.1 checkpoint does the issue violate?

## Steps to reproduce the issue

1.  go to http://react.carbondesignsystem.com/?path=/story/tooltip--default-bottom
2, use keyboard to open the tooltip (tab on the icon, and then hit enter to open）

> Please create a reduced test case in CodeSandbox
> 
> - Style and vanilla JS:
>   https://codesandbox.io/s/github/carbon-design-system/carbon/tree/master/packages/components/examples/codesandbox
> - React:
>   https://codesandbox.io/s/github/carbon-design-system/carbon/tree/master/packages/react/examples/codesandbox

## Additional information

- Screenshots or code
- Notes

**Comment by jingx-xue** [comment]
when use tab to open the tooltip, the tooltip content position is wrong.
It happens only when you using keyboard to open the tooltip, and get fixed if you click on the tooltip icon,
also happens when you resize the window and use keyboard to open the tooltip
![image](https://user-images.githubusercontent.com/48215968/60197208-f8a36c80-980c-11e9-8dbd-c86a95dd5e4d.png)

**Comment by asudoh** [comment]
Hi 👋 cannot reproduce on my end. There may be additional factors to what you described. Would you want to take a movie to put it here? Thanks!
